test/integration/site_layout_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class SiteLayoutTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+  test "layout links" do
+    get root_path
+    assert_template 'static_pages/home'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.